tests/test_bq_data_collector.py

```diff
@@ -0,0 +1,72 @@
+import unittest
+from unittest.mock import patch, MagicMock
+
+import arrow
+import pandas as pd
+
+import src.utils.cloud_constants as cc
+import tests.test_helper as test_helper
+from src.bq_data_collector import BigQueryDataCollector
+
+
+class BigDataCollectorTestCase(unittest.TestCase):
+
+    @patch('src.utils.bq_client_helper.create_github_bq_client', return_value=MagicMock())
+    def test_get_gh_event_as_data_frame(self, _mock_bq_client):
```

Comment:
You may need to have a test for empty BQ response also.

Reply:
aded empty BQ response test case